### ISSUE TITLE
API documentation debt - WinForms

### ISSUE THREAD

**Issue by carlossanlop** [dotnet/winforms#4163]
As part of the 6.0 goals, we want to finish documenting APIs that were shipped without triple slash documentation in previous .NET Core versions.

This work item is part of:
- https://github.com/dotnet/core/issues/5410 Epic: Make sure .NET libraries are continuously improved.
- https://github.com/dotnet/runtime/issues/43849 API documentation debt

We would like the WinForms team to help with this effort by addressing the APIs in the list below, which are missing some or all of their documentation.

Make sure to follow the documentation guidelines defined in the dotnet-api-docs wiki:
https://github.com/dotnet/dotnet-api-docs/wiki

Please add me as a PR reviewer.

Community contributions are welcome.

<details>
  <summary>System.Windows.Forms undocumented APIs</summary>

  DocId | Summary | Parameters | TypeParameters | ReturnValue | Source File Path | Docs URL
  -- | -- | -- | -- | -- | -- | --
  E:System.Windows.Forms.ListView.GroupCollapsedStateChanged | Missing | NA | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/ListView.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.ListView.GroupCollapsedStateChanged?branch=master
  E:System.Windows.Forms.ListView.GroupTaskLinkClick | Missing | NA | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/ListView.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.ListView.GroupTaskLinkClick?branch=master
  F:System.Windows.Forms.ListViewGroupCollapsedState.Collapsed | Missing | NA | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/ListViewGroupCollapsedState.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.ListViewGroupCollapsedState?branch=master
  F:System.Windows.Forms.ListViewGroupCollapsedState.Default | Missing | NA | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/ListViewGroupCollapsedState.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.ListViewGroupCollapsedState?branch=master
  F:System.Windows.Forms.ListViewGroupCollapsedState.Expanded | Missing | NA | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/ListViewGroupCollapsedState.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.ListViewGroupCollapsedState?branch=master
  F:System.Windows.Forms.VisualStyles.FontProperty.TextFont | Missing | NA | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms.VisualStyles/FontProperty.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.VisualStyles.FontProperty?branch=master
  M:System.Windows.Forms.Button.CreateAccessibilityInstance | Missing | NA | NA | Missing | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/Button.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.Button.CreateAccessibilityInstance?branch=master
  M:System.Windows.Forms.DomainUpDown.DomainUpDownAccessibleObject.#ctor<br />(System.Windows.Forms.DomainUpDown) | Missing | Missing | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/DomainUpDown+DomainUpDownAccessibleObject.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.DomainUpDown.DomainUpDownAccessibleObject.-ctor?branch=master
  M:System.Windows.Forms.DrawItemEventArgs.Dispose | Missing | NA | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/DrawItemEventArgs.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.DrawItemEventArgs.Dispose?branch=master
  M:System.Windows.Forms.DrawItemEventArgs.Dispose(System.Boolean) | Missing | Missing | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/DrawItemEventArgs.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.DrawItemEventArgs.Dispose?branch=master
  M:System.Windows.Forms.DrawItemEventArgs.System#Drawing#IDeviceContext#GetHdc | Missing | NA | NA | Missing | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/DrawItemEventArgs.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.DrawItemEventArgs.System-Drawing-IDeviceContext-GetHdc?branch=master
  M:System.Windows.Forms.DrawItemEventArgs.System#Drawing#IDeviceContext#ReleaseHdc | Missing | NA | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/DrawItemEventArgs.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.DrawItemEventArgs.System-Drawing-IDeviceContext-ReleaseHdc?branch=master
  M:System.Windows.Forms.HScrollBar.CreateAccessibilityInstance | Missing | NA | NA | Missing | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/HScrollBar.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.HScrollBar.CreateAccessibilityInstance?branch=master
  M:System.Windows.Forms.ListBox.ObjectCollection.System#Collections#IList#Contains<br />(System.Object) | Missing | Missing | NA | Missing | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/ListBox+ObjectCollection.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.ListBox.ObjectCollection.System-Collections-IList-Contains?branch=master
  M:System.Windows.Forms.ListBox.ObjectCollection.System#Collections#IList#IndexOf<br />(System.Object) | Missing | Missing | NA | Missing | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/ListBox+ObjectCollection.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.ListBox.ObjectCollection.System-Collections-IList-IndexOf?branch=master
  M:System.Windows.Forms.ListBox.ObjectCollection.System#Collections#IList#Insert<br />(System.Int32,System.Object) | Missing | Missing | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/ListBox+ObjectCollection.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.ListBox.ObjectCollection.System-Collections-IList-Insert?branch=master
  M:System.Windows.Forms.ListBox.ObjectCollection.System#Collections#IList#Remove<br />(System.Object) | Missing | Missing | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/ListBox+ObjectCollection.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.ListBox.ObjectCollection.System-Collections-IList-Remove?branch=master
  M:System.Windows.Forms.ListView.OnGroupCollapsedStateChanged<br />(System.Windows.Forms.ListViewGroupEventArgs) | Missing | Missing | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/ListView.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.ListView.OnGroupCollapsedStateChanged?branch=master
  M:System.Windows.Forms.ListView.OnGroupTaskLinkClick(System.Windows.Forms.ListViewGroupEventArgs) | Missing | Missing | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/ListView.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.ListView.OnGroupTaskLinkClick?branch=master
  M:System.Windows.Forms.ListViewGroupEventArgs.#ctor(System.Int32) | Missing | Missing | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/ListViewGroupEventArgs.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.ListViewGroupEventArgs.-ctor?branch=master
  M:System.Windows.Forms.PaintEventArgs.System#Drawing#IDeviceContext#GetHdc | Missing | NA | NA | Missing | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/PaintEventArgs.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.PaintEventArgs.System-Drawing-IDeviceContext-GetHdc?branch=master
  M:System.Windows.Forms.PaintEventArgs.System#Drawing#IDeviceContext#ReleaseHdc | Missing | NA | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/PaintEventArgs.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.PaintEventArgs.System-Drawing-IDeviceContext-ReleaseHdc?branch=master
  M:System.Windows.Forms.PictureBox.CreateAccessibilityInstance | Missing | NA | NA | Missing | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/PictureBox.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.PictureBox.CreateAccessibilityInstance?branch=master
  M:System.Windows.Forms.RichTextBox.CreateAccessibilityInstance | Missing | NA | NA | Missing | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/RichTextBox.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.RichTextBox.CreateAccessibilityInstance?branch=master
  M:System.Windows.Forms.ScrollBar.CreateAccessibilityInstance | Missing | NA | NA | Missing | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/ScrollBar.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.ScrollBar.CreateAccessibilityInstance?branch=master
  M:System.Windows.Forms.Splitter.CreateAccessibilityInstance | Missing | NA | NA | Missing | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/Splitter.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.Splitter.CreateAccessibilityInstance?branch=master
  M:System.Windows.Forms.TaskDialogButton.#ctor<br />(System.String,System.Boolean,System.Boolean) | Present | Missing | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TaskDialogButton.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TaskDialogButton.-ctor?branch=master
  M:System.Windows.Forms.TaskDialogButton.Equals(System.Object) | Missing | Missing | NA | Missing | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TaskDialogButton.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TaskDialogButton.Equals?branch=master
  M:System.Windows.Forms.TaskDialogButton.GetHashCode | Missing | NA | NA | Missing | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TaskDialogButton.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TaskDialogButton.GetHashCode?branch=master
  M:System.Windows.Forms.TaskDialogButton.op_Equality<br />(System.Windows.Forms.TaskDialogButton,System.Windows.Forms.TaskDialogButton) | Missing | Missing | NA | Missing | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TaskDialogButton.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TaskDialogButton.op_Equality?branch=master
  M:System.Windows.Forms.TaskDialogButton.op_Inequality<br />(System.Windows.Forms.TaskDialogButton,System.Windows.Forms.TaskDialogButton) | Missing | Missing | NA | Missing | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TaskDialogButton.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TaskDialogButton.op_Inequality?branch=master
  M:System.Windows.Forms.TaskDialogButtonCollection.ClearItems | Missing | NA | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TaskDialogButtonCollection.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TaskDialogButtonCollection.ClearItems?branch=master
  M:System.Windows.Forms.TaskDialogButtonCollection.InsertItem<br />(System.Int32,System.Windows.Forms.TaskDialogButton) | Missing | Missing | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TaskDialogButtonCollection.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TaskDialogButtonCollection.InsertItem?branch=master
  M:System.Windows.Forms.TaskDialogButtonCollection.RemoveItem(System.Int32) | Missing | Missing | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TaskDialogButtonCollection.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TaskDialogButtonCollection.RemoveItem?branch=master
  M:System.Windows.Forms.TaskDialogButtonCollection.SetItem<br />(System.Int32,System.Windows.Forms.TaskDialogButton) | Missing | Missing | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TaskDialogButtonCollection.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TaskDialogButtonCollection.SetItem?branch=master
  M:System.Windows.Forms.TaskDialogCommandLinkButton.#ctor<br />(System.String,System.String,System.Boolean,System.Boolean) | Present | Missing | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TaskDialogCommandLinkButton.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TaskDialogCommandLinkButton.-ctor?branch=master
  M:System.Windows.Forms.TaskDialogFootnote.op_Implicit<br />(System.String)~System.Windows.Forms.TaskDialogFootnote | Missing | Missing | NA | Missing | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TaskDialogFootnote.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TaskDialogFootnote.op_Implicit?branch=master
  M:System.Windows.Forms.TaskDialogRadioButton.#ctor(System.String) | Present | Missing | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TaskDialogRadioButton.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TaskDialogRadioButton.-ctor?branch=master
  M:System.Windows.Forms.TaskDialogRadioButtonCollection.#ctor | Missing | NA | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TaskDialogRadioButtonCollection.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TaskDialogRadioButtonCollection.-ctor?branch=master
  M:System.Windows.Forms.TaskDialogRadioButtonCollection.Add(System.String) | Missing | Missing | NA | Missing | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TaskDialogRadioButtonCollection.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TaskDialogRadioButtonCollection.Add?branch=master
  M:System.Windows.Forms.TaskDialogRadioButtonCollection.ClearItems | Missing | NA | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TaskDialogRadioButtonCollection.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TaskDialogRadioButtonCollection.ClearItems?branch=master
  M:System.Windows.Forms.TaskDialogRadioButtonCollection.InsertItem<br />(System.Int32,System.Windows.Forms.TaskDialogRadioButton) | Missing | Missing | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TaskDialogRadioButtonCollection.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TaskDialogRadioButtonCollection.InsertItem?branch=master
  M:System.Windows.Forms.TaskDialogRadioButtonCollection.RemoveItem(System.Int32) | Missing | Missing | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TaskDialogRadioButtonCollection.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TaskDialogRadioButtonCollection.RemoveItem?branch=master
  M:System.Windows.Forms.TaskDialogRadioButtonCollection.SetItem<br />(System.Int32,System.Windows.Forms.TaskDialogRadioButton) | Missing | Missing | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TaskDialogRadioButtonCollection.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TaskDialogRadioButtonCollection.SetItem?branch=master
  M:System.Windows.Forms.TaskDialogVerificationCheckBox.op_Implicit<br />(System.String)~System.Windows.Forms.TaskDialogVerificationCheckBox | Missing | Missing | NA | Missing | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TaskDialogVerificationCheckBox.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TaskDialogVerificationCheckBox.op_Implicit?branch=master
  M:System.Windows.Forms.TextBox.OnKeyUp(System.Windows.Forms.KeyEventArgs) | Missing | Missing | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TextBox.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TextBox.OnKeyUp?branch=master
  M:System.Windows.Forms.TextBox.OnMouseDown(System.Windows.Forms.MouseEventArgs) | Missing | Missing | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TextBox.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TextBox.OnMouseDown?branch=master
  M:System.Windows.Forms.TextBoxBase.CreateAccessibilityInstance | Missing | NA | NA | Missing | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TextBoxBase.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TextBoxBase.CreateAccessibilityInstance?branch=master
  M:System.Windows.Forms.TextRenderer.DrawText<br />(System.Drawing.IDeviceContext,System.ReadOnlySpan{System.Char},<br />System.Drawing.Font,System.Drawing.Point,System.Drawing.Color) | Missing | Missing | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TextRenderer.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TextRenderer.DrawText?branch=master
  M:System.Windows.Forms.TextRenderer.DrawText<br />(System.Drawing.IDeviceContext,System.ReadOnlySpan{System.Char},<br />System.Drawing.Font,System.Drawing.Point,System.Drawing.Color,System.Drawing.Color) | Missing | Missing | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TextRenderer.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TextRenderer.DrawText?branch=master
  M:System.Windows.Forms.TextRenderer.DrawText<br />(System.Drawing.IDeviceContext,System.ReadOnlySpan{System.Char},<br />System.Drawing.Font,System.Drawing.Point,System.Drawing.Color,<br />System.Drawing.Color,System.Windows.Forms.TextFormatFlags) | Missing | Missing | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TextRenderer.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TextRenderer.DrawText?branch=master
  M:System.Windows.Forms.TextRenderer.DrawText<br />(System.Drawing.IDeviceContext,System.ReadOnlySpan{System.Char},<br />System.Drawing.Font,System.Drawing.Point,System.Drawing.Color,System.Windows.Forms.TextFormatFlags) | Missing | Missing | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TextRenderer.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TextRenderer.DrawText?branch=master
  M:System.Windows.Forms.TextRenderer.DrawText<br />(System.Drawing.IDeviceContext,System.ReadOnlySpan{System.Char},<br />System.Drawing.Font,System.Drawing.Rectangle,System.Drawing.Color) | Missing | Missing | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TextRenderer.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TextRenderer.DrawText?branch=master
  M:System.Windows.Forms.TextRenderer.DrawText<br />(System.Drawing.IDeviceContext,System.ReadOnlySpan{System.Char},<br />System.Drawing.Font,System.Drawing.Rectangle,System.Drawing.Color,System.Drawing.Color) | Missing | Missing | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TextRenderer.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TextRenderer.DrawText?branch=master
  M:System.Windows.Forms.TextRenderer.DrawText<br />(System.Drawing.IDeviceContext,System.ReadOnlySpan{System.Char},<br />System.Drawing.Font,System.Drawing.Rectangle,System.Drawing.Color,<br />System.Drawing.Color,System.Windows.Forms.TextFormatFlags) | Missing | Missing | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TextRenderer.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TextRenderer.DrawText?branch=master
  M:System.Windows.Forms.TextRenderer.DrawText<br />(System.Drawing.IDeviceContext,System.ReadOnlySpan{System.Char},<br />System.Drawing.Font,System.Drawing.Rectangle,System.Drawing.Color,System.Windows.Forms.TextFormatFlags) | Missing | Missing | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TextRenderer.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TextRenderer.DrawText?branch=master
  M:System.Windows.Forms.TextRenderer.MeasureText<br />(System.Drawing.IDeviceContext,System.ReadOnlySpan{System.Char},<br />System.Drawing.Font) | Missing | Missing | NA | Missing | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TextRenderer.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TextRenderer.MeasureText?branch=master
  M:System.Windows.Forms.TextRenderer.MeasureText<br />(System.Drawing.IDeviceContext,System.ReadOnlySpan{System.Char},<br />System.Drawing.Font,System.Drawing.Size) | Missing | Missing | NA | Missing | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TextRenderer.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TextRenderer.MeasureText?branch=master
  M:System.Windows.Forms.TextRenderer.MeasureText<br />(System.Drawing.IDeviceContext,System.ReadOnlySpan{System.Char},<br />System.Drawing.Font,System.Drawing.Size,System.Windows.Forms.TextFormatFlags) | Missing | Missing | NA | Missing | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TextRenderer.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TextRenderer.MeasureText?branch=master
  M:System.Windows.Forms.TextRenderer.MeasureText<br />(System.ReadOnlySpan{System.Char},System.Drawing.Font) | Missing | Missing | NA | Missing | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TextRenderer.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TextRenderer.MeasureText?branch=master
  M:System.Windows.Forms.TextRenderer.MeasureText<br />(System.ReadOnlySpan{System.Char},System.Drawing.Font,System.Drawing.Size) | Missing | Missing | NA | Missing | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TextRenderer.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TextRenderer.MeasureText?branch=master
  M:System.Windows.Forms.TextRenderer.MeasureText<br />(System.ReadOnlySpan{System.Char},System.Drawing.Font,System.Drawing.Size,<br />System.Windows.Forms.TextFormatFlags) | Missing | Missing | NA | Missing | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TextRenderer.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TextRenderer.MeasureText?branch=master
  M:System.Windows.Forms.VScrollBar.CreateAccessibilityInstance | Missing | NA | NA | Missing | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/VScrollBar.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.VScrollBar.CreateAccessibilityInstance?branch=master
  P:System.Windows.Forms.ListBox.ObjectCollection.System#Collections#IList#Item<br />(System.Int32) | Missing | Missing | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/ListBox+ObjectCollection.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.ListBox.ObjectCollection.System-Collections-IList-Item?branch=master
  P:System.Windows.Forms.ListView.GroupImageList | Missing | NA | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/ListView.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.ListView.GroupImageList?branch=master
  P:System.Windows.Forms.ListViewGroup.CollapsedState | Missing | NA | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/ListViewGroup.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.ListViewGroup.CollapsedState?branch=master
  P:System.Windows.Forms.ListViewGroup.Subtitle | Missing | NA | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/ListViewGroup.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.ListViewGroup.Subtitle?branch=master
  P:System.Windows.Forms.ListViewGroup.TaskLink | Missing | NA | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/ListViewGroup.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.ListViewGroup.TaskLink?branch=master
  P:System.Windows.Forms.ListViewGroup.TitleImageIndex | Missing | NA | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/ListViewGroup.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.ListViewGroup.TitleImageIndex?branch=master
  P:System.Windows.Forms.ListViewGroup.TitleImageKey | Missing | NA | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/ListViewGroup.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.ListViewGroup.TitleImageKey?branch=master
  P:System.Windows.Forms.ListViewGroupEventArgs.GroupIndex | Missing | NA | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/ListViewGroupEventArgs.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.ListViewGroupEventArgs.GroupIndex?branch=master
  T:System.Windows.Forms.ListViewGroupCollapsedState | Missing | NA | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/ListViewGroupCollapsedState.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.ListViewGroupCollapsedState?branch=master
  T:System.Windows.Forms.ListViewGroupEventArgs | Missing | NA | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/ListViewGroupEventArgs.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.ListViewGroupEventArgs?branch=master
  T:System.Windows.Forms.TaskDialogRadioButtonCollection | Missing | NA | NA | NA | https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/TaskDialogRadioButtonCollection.xml | https://review.docs.microsoft.com/en-us/dotnet/api/System.Windows.Forms.TaskDialogRadioButtonCollection?branch=master

</details>

cc @merriemcgaw

**Comment by RussKie** [comment]
@carlossanlop a number of API that are listed in your report are annotated, and have been since their addition to the codebase. E.g. almost all of the `ListView` and `TaskDialog` related are false positives - those API have been extensively annotated at the time we accepted respective PRs (#3155 and #1133).

I would think that our xml-docs weren't correctly ingested into the docs repo, which led to this. In the interest of time can you please arrange ingestion of the latest 5.0 build artifacts, so we can concentrate only on the missing docs.
Thank you.

**Comment by carlossanlop** [comment]
Thank you for the information, @Russkie . The PRs #3155, #1133 and #3711 merged triple slash comments but haven't yet been ported to dotnet-api-docs. You can use [DocsPortingTool ](https://github.com/carlossanlop/DocsPortingTools) to do so, then submit a PR. After that is merged, I will be able to get an updated report with the remaining undoc APIs in WinForms.

If you get a chance to do the porting yourself, that would be great. If not, let me know, I could eventually do it myself, but not right away.

**Comment by AdamYoblick** [comment]
Thanks @carlossanlop , I'll take a look at this and let you know if I get stuck 😄

**Comment by AdamYoblick** [comment]
@carlossanlop I've run the following command line:

```
.\DocsPortingTool\bin\Debug\net5.0\DocsPortingTool.exe -Docs E:\src\repos\github\dotnet-api-docs\xml -TripleSlash E:\src\repos\github\winforms\artifacts\bin -IncludedAssemblies Accessibility,Microsoft.VisualBasic.Forms,System.Windows.Forms.Design,System.Windows.Forms.Primitives,System.Windows.Forms -Save true
```

I'm running into an issue that I'm not sure how to fix through the tool. The error is this:

```
Problem in param 'enabled' in member 'M:System.Windows.Forms.TaskDialogButton.#ctor(System.String,System.Boolean,System.Boolean)' in file 'E:\src\repos\github\dotnet-api-docs\xml\System.Windows.Forms\TaskDialogButton.xml'
The param probably exists in code, but the exact name was not found in Docs. What would you like to do?
    0 - Exit program.
    1 - Select the correct triple slash param from the existing ones.
    2 - Ignore this param.
      Note:Make sure to double check the affected Docs file after the tool finishes executing.
Your answer [0,1,2]: 1
Triple slash params found in member 'M:System.Windows.Forms.TaskDialogButton.#ctor(System.String,System.Boolean,System.Boolean)':
    0 - Exit program.
    1 - text
    2 - allowCloseDialog
Your answer to match param 'enabled'? [0..2]: 0
```

I checked the code and it looks like a param called `enabled` is in the constructor. I would have expected the tool to add this param to the docs, but that doesn't seem to be a choice here. It only lets me choose an existing param from the existing ones, or ignore this param. Is there no way to ADD this param to the docs, or am I misunderstanding what's going on here?

**Comment by RussKie** [comment]
@AdamYoblick I think this indicates an omission in our code - we missing docs for `enabled`:
https://github.com/dotnet/winforms/blob/1d8dba7ff63e19b5784913ea773635f0ded0c5a8/src/System.Windows.Forms/src/System/Windows/Forms/TaskDialogButton.cs#L67-L77

**Comment by AdamYoblick** [comment]
Ah ok, makes sense. Igor, should I ignore all of these just to get the docs updated, or should I fix them up at the same time?

________________________________

**Comment by RussKie** [comment]
Would be good to fix it - it will both unblock you in this exercise, and we'll get us to the "0" in the docs register :) Holler, if you need a hand.

**Comment by AdamYoblick** [comment]
Will do Igor. I'll run the docs tool to find all the missing triple slash comments, fix them, and get them into a PR into this repo. Once everyone is happy with that text, then I'll re-generate the docs and create a PR into the docs repo.

**Comment by AdamYoblick** [comment]
@RussKie @JeremyKuhne There are lots of triple slash docs missing from methods in TextRenderer.cs. Most of them are on overloads. I will fix up methods that are missing specific params, but I think we need a separate task to go back and fix large blocks of comments.

**Comment by carlossanlop** [comment]
@AdamYoblick sorry for the late reply. I believe @Russkie gave you the answer. The tool is able to help you figure out what the right parameter is, based on the information it finds on both the intellisense xml and the dotnet-api-docs xml.
Feel free to reach out to me directly on Teams if you need a more immediate answer.

**Comment by JeremyKuhne** [comment]
> There are lots of triple slash docs missing from methods in TextRenderer.cs.

@AdamYoblick I added the XML comments for the _new_ overloads- the existing ones all have documentation so I didn't bother adding new XML comments there. For the new overloads (Span) I just copied stuff from docs and tweaked where needed.

**Comment by AdamYoblick** [comment]
I merged in the PR to fix most of the debt. We still have some debt in the form of suggestions from the docs team, which is tracked with #4194
